src/components/checkbox/CheckBox.svelte

```diff
@@ -0,0 +1,122 @@
+<label
```

Comment:
This is nitpicky but can you rename the Checkbox files so the 'B' in Checkbox is not capitalized?